sample-scenarios/sample8/src/main/java/CreateSampleEightRawData.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.wso2.carbon.apimgt.samples.utils.SampleUtils;
+import org.wso2.carbon.apimgt.samples.utils.TenantUtils;
+import org.wso2.carbon.apimgt.samples.utils.publisher.rest.client.ApiException;
+import org.wso2.carbon.apimgt.samples.utils.publisher.rest.client.ApiResponse;
+import org.wso2.carbon.apimgt.samples.utils.publisher.rest.client.model.API;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class is used to populate sample data to represent the sample eight business scenario.
+ */
+public class CreateSampleEightRawData {
+
+    private static final String hostname = "localhost";
+    private static final String port = "9443";
+
+    /**
+     * This main method will be called when running sample one.
```

Comment:
should be sample 8 right? :)

Reply:
Thanks for pointing out. FIxed in e29065f4d11f91950e1f30d35d5cfdc1acdf6655